mlflow.yaml

```diff
@@ -49,6 +49,8 @@ pipeline:
       # Create virtual environment
       python -m venv venv --system-site-packages
       source venv/bin/activate
+      # Bump setuptools to address CVE
```

Comment:
Looks like it didn't work:

```bash
🔎 Scanning "/tmp/artifacts-1/packages/x86_64/mlflow-bitnami-2.16.1-r0.apk"
└── 📄 /usr/share/mlflow/lib/python3.12/site-packages/virtualenv/seed/wheels/embed/setuptools-68.0.0-py3-none-any.whl
        📦 setuptools 68.0.0 (python)
            High CVE-20[24](https://github.com/wolfi-dev/os/actions/runs/10865226677/job/30151594564?pr=28372#step:9:25)-6345 GHSA-cx63-2mw6-8hw5 fixed in 70.0.0
```

Reply:
Are you still working on this @kranurag7?

Reply:
Mark I tried it for a very short time this morning. With this patch, it's not working out. 

```diff
-      pip install -I packaging pyparsing setuptools --no-compile
+      pip install -I packaging pyparsing setuptools==70.0.0 --no-compile
```
I'm not working on this as of now.